src/strategies/orange-reputation-based-voting/index.ts

```diff
@@ -0,0 +1,39 @@
+import fetch from 'cross-fetch';
+
+export const author = 'orange-protocol';
+export const version = '0.1.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
```

Comment:
'options' is defined but never used 🤔 but I see address and decimals being passed from examples.json?